internal/cli/shim.go

```
@@ -175,38 +177,105 @@
+	log.Debug().Any("pty", ptmx.Fd()).Any("tty", tty.Fd()).Msg("pty created")
 	defer func() {
 		if err := tty.Close(); err != nil {
 			log.Error().Err(err).Msg("close tty")
 		}
-	}() // Best effort.
+	}()
+	conns := &multiwriter{}
+	ptmxr := io.TeeReader(ptmx, conns)
 	go func() {
-		if _, err := io.Copy(outw, ptmx); err != nil {
+		if _, err := io.Copy(outw, ptmxr); err != nil {
```

golangci-lint (1.62.2)

shadow: declaration of "err" shadows declaration at line 225

(govet)